mne/viz/_brain/_timeviewer.py

```diff
@@ -1199,28 +1200,49 @@ def clean(self):
 class _LinkViewer(object):
     """Class to link multiple _TimeViewer objects."""
 
-    def __init__(self, brains):
+    def __init__(self, brains, time=True, camera=False):
         self.brains = brains
         self.time_viewers = [brain.time_viewer for brain in brains]
 
-        # link time sliders
-        self.link_sliders(
-            name="_time_slider",
-            callback=self.set_time_point,
-            event_type="always"
-        )
+        # check time infos
+        brain_times = [brain._times for brain in brains]
+        if not all(len(x) == len(brain_times[0]) for x in brain_times):
+            time = False
+        if not all(all(np.equal(x, brain_times[0])) for x in brain_times):
+            time = False
+
```

Comment:
This second check throws `ValueError: operands could not be broadcast` if the `brain_times` are of different lengths and more than one.

why not only do the check if you have to (`time == True`)

```
        if time and not all(len(x) == len(brain_times[0]) for x in brain_times):
            time = False
        if time and not all(all(np.equal(x, brain_times[0])) for x in brain_times):
            time = False
```

to test just link a plot of an evoked `stc` with a cropped copy of the same `stc`